### PR TITLE
Clarify guidelines around use of role pings in discord

### PR DESCRIPTION
**Pull request by optimumtact**
This apparently  is not clear to people.
cc @Mothblocks @ninjanomnom @MrStonedOne

### CODE REVIEW

**Comment by Ryll-Ryll** [comment]
Requires another grammatical pass

**Review by Ryll-Ryll — CHANGES_REQUESTED** [review]
Requires another grammar pass

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -191,3 +191,9 @@ Unless overridden or a non standard git binary is used the line ending settings
 Note: VSC requires an [extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) to take advantage of editorconfig.
 
 Github actions that require additional configuration are disabled on the repository until ACTION_ENABLER secret is created with non-empty value.
+
+## Using the maintainer role ping in discord
```
Comment (san7890) [inline .github/CONTRIBUTING.md:195]:
```suggestion
## Using the Maintainer Role Ping in Discord
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -191,3 +191,9 @@ Unless overridden or a non standard git binary is used the line ending settings
...
+
+This role is pingable as a compromise reached with the server host mso over the auto staling system. It should be used only to ping maintainers when your PR has had the stale label applied. Using it before then can be met with escalating timeouts and referall to the discord moderators for further infractions.
```
Comment (san7890) [inline .github/CONTRIBUTING.md:197]:
```suggestion
This role `@Maintainer` is pingable as a compromise reached with the server host MrStonedOne over the auto-staling system we presently have in the codebase. It should be used only to ping Maintainers when your PR has had the "Stale" label applied. Using it before then can be met with escalating timeouts and referral to /tg/station's Discord moderators for further infractions.
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -191,3 +191,9 @@ Unless overridden or a non standard git binary is used the line ending settings
...
+
+Feel free to engage in general in coding general without the role ping, before your PR goes stale to build interest and get reviews.
```
Comment (san7890) [inline .github/CONTRIBUTING.md:199]:
```suggestion
Feel free to engage and obtain general feedback in the Coding General channel without the role ping before your PR goes stale to build interest and get reviews.
```

**Review by san7890 — COMMENTED** [review]
grammar pass i guess

feel free to disregard all of this if it's not what you're looking for

**Review by Mothblocks — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -191,3 +191,9 @@ Unless overridden or a non standard git binary is used the line ending settings
 Note: VSC requires an [extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) to take advantage of editorconfig.
 
 Github actions that require additional configuration are disabled on the repository until ACTION_ENABLER secret is created with non-empty value.
+
+## Using the Maintainer Role Ping in Discord
+
+This role `@Maintainer` is pingable as a compromise reached with the server host MrStonedOne over the auto-staling system we presently have in the codebase. It should be used only to ping Maintainers when your PR has had the "Stale" label applied. Using it before then can be met with escalating timeouts and referral to /tg/station's Discord moderators for further infractions.
```
Comment (MrStonedOne) [inline .github/CONTRIBUTING.md:197]:
```suggestion
This role `@Maintainer` is pingable as a compromise reached with the server host MrStonedOne over the auto-stale system we presently have in the codebase. It should be used only to ping Maintainers when your PR has had the "Stale" label applied. Using it before then can be met with escalating timeouts and referral to /tg/station's Discord moderators for further infractions.
```

staling is more grammatically correct, but even after a nights sleep it kept reading it as scaling or stalling.

maybe my brain just hates the word stale.

**Review by MrsTonedOne — APPROVED** [review]

**Pull request merged**